scripts/launch_server.py

```diff
@@ -45,6 +45,21 @@ def launch() -> None:
     logger.info(' '.join(redis_cmd))
     subprocess.Popen(redis_cmd)
 
+    # launch the python server if bot option is true
+    bot = 'bots'
+    if bot in config and config[bot]:
+        py_command = ['python3.6', '-m', 'scalabel.server']
```

Comment:
python3.6 may be too old for our code base. 3.7 and 3.8 are ok. Or just put python3 here.

Reply:
The issue is that the polyrnn codebase uses tensorflow 1.3, which is not supported past python 3.6.